### PR TITLE
Feat/introduction

### PR DESCRIPTION
**Pull request by jyng**

### CODE REVIEW

**Comment by urcades** [comment]
@g-a-v-i-n @matildepark do either of ya have any idea why our checks might be failing at this point? I'm getting a possibly related error while trying to start up zola:

Screen Shot 2019-10-10 at 8 22 56 PM


It's kind of crazy because neither I nor @jyng touched anything in docs afaik

**Comment by matildepark** [comment]
Working with Jimmy on it out of band. As with Gavin’s PR, it comes from a relative link not existing. I committed to docs a few hours ago and pulled docs into the branch. So merge next-slim back in.

File: templates/understanding-urbit/list.html [hunk 1]
```diff
@@ -8,65 +8,39 @@
   <a href="/understanding-urbit/">
     Understanding Urbit
   </a>
-  {% include "partials/navigation-search.html" %}
```
Comment (matildepark) [inline templates/understanding-urbit/list.html:11]: Did you intend to remove the Search from the sidebar on UU's intro, as well? And the sidebar is on the other pages — if you want to remove it from all of them, you'll have to do that again in `/templates/understanding-urbit/post.html`

**Review by matildepark — APPROVED** [review]
Otherwise this is good to go.

**Comment by jyng** [comment]
its intentional for introduction to not have a sidebar so people can focus on the page. LGTM.

**Pull request merged**